inference-engine/tests/functional/plugin/cpu/single_layer_tests/scatter_ND_update.cpp

```diff
@@ -0,0 +1,177 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "functional_test_utils/ov_tensor_utils.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
+#include "ngraph_functions/builders.hpp"
+
+using namespace ngraph;
+using namespace InferenceEngine;
+using namespace CPUTestUtils;
+using namespace ov::test;
+
+namespace CPULayerTestsDefinitions {
+using ScatterNDUpdateShapes = std::vector<InputShape>;
+using IndicesValues = std::vector<std::int64_t>;
+
+struct ScatterNDUpdateLayerParams {
+    ScatterNDUpdateShapes inputShapes;
+    IndicesValues indicesValues;
+};
+
+using scatterUpdateParams = std::tuple<
+    ScatterNDUpdateLayerParams,
+    ElementType,        // input precision
+    ElementType>;       // indices precision
+
+class ScatterNDUpdateLayerCPUTest : public testing::WithParamInterface<scatterUpdateParams>, public SubgraphBaseTest, public CPUTestsBase {
+public:
+    static std::string getTestCaseName(testing::TestParamInfo<scatterUpdateParams> obj) {
+        ScatterNDUpdateLayerParams scatterParams;
+        ElementType inputPrecision;
+        ElementType idxPrecision;
+        std::tie(scatterParams, inputPrecision, idxPrecision) = obj.param;
+        const auto inputShapes = scatterParams.inputShapes;
+        const auto indicesValues = scatterParams.indicesValues;
+
+        std::ostringstream result;
+        result << inputPrecision << "_IS=";
+        for (const auto& shape : inputShapes) {
+            result << CommonTestUtils::partialShape2str({ shape.first }) << "_";
+        }
+        result << "TS=";
+        for (const auto& shape : inputShapes) {
+            result << "(";
+            if (!shape.second.empty()) {
+                auto itr = shape.second.begin();
+                do {
```

Comment:
minor
here and in other tests maybe better to use `for`?

Reply:
Done, thanks!